ddtrace/appsec/gateway.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+from typing import Set
+from typing import TYPE_CHECKING
+
+import attr
+
+
+if TYPE_CHECKING:
+    from ddtrace.span import _RequestStore
+
+
+class _Addresses(Enum):
```

Comment:
```suggestion
class _Addresses(object):
```

enums are a little clunky to deal with.